src/tools/rust-analyzer/crates/base-db/Cargo.toml

```diff
@@ -30,5 +30,9 @@ vfs.workspace = true
 span.workspace = true
 intern.workspace = true
 
+[features]
+default = []
+in-rust-tree = []
```

Comment:
Maybe could be `--cfg` instead but idk

Reply:
I was hoping we could make this a cfg at some point. Either way out of scope for this PR

Reply:
It probably can, I just yolo'd it as cargo features to get it to build first of all.